File: app/src/main/java/com/kshitijchauhan/haroldadmin/moviedb/Genre.java
package com.kshitijchauhan.haroldadmin.moviedb;

public class Genre {

    private String id;
    private String name;

    public Genre(String name) {
        this.name = name;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getId() {
        return id;
    }
}

File: tmdb_repository/src/main/java/com/haroldadmin/tmdb_repository/movie/local/models/Genre.kt
package com.haroldadmin.tmdb_repository.movie.local.models

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "genres")
data class Genre(
    @PrimaryKey
    val id: Int,
    @ColumnInfo(name = "name")
    val name: String
)